tests/test_pipeline_config_generator.py

```diff
@@ -0,0 +1,43 @@
+import generators.preprocessor.pipeline_generator as pipeline_generator
+import os
+import sys
+sys.path.append(os.getcwd())
```

Comment:
```suggestion
import os
import sys
sys.path.append(os.getcwd())

import generators.preprocessor.pipeline_generator as pipeline_generator
```

Had to change the order, minor fix